tests/bin/rpminspect.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+# Don't run metadata check as we can't know the build host subdomain
+# of CI runners in advance to add to an allow list
+
+echo "Running RPMInspect on SRPM"
+rpminspect-fedora -E metadata /github/home/build/jss/SRPMS/*.rpm
```

Comment:
After looking tomcatjss PR I agree this (and line 10) could be relative path for a more usability of the script

Reply:
Ack, will update, thanks!